src/components/UserProfile/UserIntegrations.js

```diff
@@ -0,0 +1,225 @@
+import CavaticaInput from 'components/UserProfile/CavaticaTokenInput';
+import gen3Logo from 'assets/logo-gen3-data-commons.svg';
+import cavaticaLogo from 'assets/logo-cavatica.svg';
+import { CAVATICA, GEN3 } from 'common/constants';
+import { getUser } from 'services/cavatica';
+
+const styles = css`
+  table {
+    border-collapse: collapse;
+  }
+  td,
+  th {
+    padding: 12px;
+    font-weight: normal;
+  }
+  td {
+    background: white;
+  }
+  font-size: 14px;
+  span {
+    flex: 1;
+  }
+  button {
+    flex: 1;
+    font-weight: bold;
+    padding: 6px;
+    margin: 2px;
+    padding-left: 5px;
+    text-transform: uppercase;
+  }
+
+  .logoImg {
+    width: 150px;
+  }
+
+  span.integrationHeader {
+    font-weight: bold;
+  }
+  div.integrationCell {
+    display: flex;
+    width: 100%;
+  }
+  div.integrationCell button {
+    text-transform: uppercase;
+    flex: 1;
+  }
+  .right {
+    text-align: right;
+  }
+  .connectedButton {
+    color: black;
+    background: none;
+  }
+`;
+
+const enhance = compose(
+  injectState,
+  withTheme,
+  withState('editingCavatica', 'setEditingCavatica', false),
+);
+
+const gen3Status = ({ gen3Key }) => {
+  return (
+    <div>
+      <span>Connected: {gen3Key} </span>
+    </div>
+  );
+};
+
+const cavaticaStatus = ({ theme, cavaticaKey, onEdit, onRemove }) => {
+  return (
+    <div css="flex-direction: column;">
+      <div
+        css={`
+          color: ${theme.active};
+          padding: 10px;
+        `}
+      >
+        <CheckIcon size={20} />
+        <span> Connected</span>
+      </div>
+      <div css="display: flex;">
+        <Button onClick={onEdit} className="connectedButton">
+          <PencilIcon />Edit
+        </Button>
+        <Button onClick={onRemove} className="connectedButton">
+          <XIcon />Remove
+        </Button>
+      </div>
+    </div>
+  );
+};
+
+const testMethod = () => {
+  console.log(getUser());
+};
+
+const UserIntegrations = ({ state: { integrationTokens }, effects, theme, ...props }) => {
+  return (
+    <div css={styles}>
+      <table
+        css={`
+          td,
+          th {
+            border: 1px solid ${theme.greyScale5};
+          }
+        `}
+      >
+        <thead
+          css={`
+            background: ${theme.greyScale6};
+            color: ${theme.secondary};
+            text-align: left;
+          `}
+        >
+          <tr>
+            <th>Service</th>
+            <th>Purpose</th>
+            <th>Integrate</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>
+              <img className="logoImg" src={gen3Logo} alt="Gen3 Logo" />
+            </td>
+            <td>
+              <span className="integrationHeader">Download Controlled Data</span>
+              <p>
+                Access and download controlled data by connecting your Kids First account to{' '}
+                <a href="https://gen3.kids-first.io/" rel="noopener noreferrer" target="_blank">
```

Comment:
if you use this `uikit/ExternalLink.js` it'll add the rel, target and have an external link icon